Framework/WorkflowAlgorithms/src/ExtractQENSMembers.cpp

```diff
@@ -45,8 +46,13 @@ const std::string ExtractQENSMembers::summary() const {
  */
 void ExtractQENSMembers::init() {
   declareProperty(
-      make_unique<WorkspaceProperty<>>("InputWorkspace", "", Direction::Input),
-      "The input workspace used in the fit.");
+      make_unique<WorkspaceProperty<>>("InputWorkspace", "", Direction::Input,
+                                       PropertyMode::Optional),
+      "The input workspace used in the fit. Ignored if 'InputWorkspaces' "
+      "property is provided.");
+  declareProperty(
```

Comment:
It is only currently used by the Python interfaces and indirectly through the ExtractMembers of property of ConvolutionFitSequential.

Although, I'm not sure I understand how this might break the Python interface, since the InputWorkspace property can be provided, while the InputWorkspaces property is left empty and the ExtractQENSMembers algorithm should behave in the same manner as it did prior to the change.

Reply:
Adding a property in the middle will break existing scripts which don't use named arguments, e.g.
```python
ExtractQENSMembers('input_ws', 'result_ws', true, ['a', 'b'], OutputWorkspace='output_ws')
```
will not work after these changes.

The question is, if some user has such scripts (maybe check our algorithm usage database?), then new properties should be appended to the end of the existing properties, i.e. declared last in `init()`.